libs/WordPressAnnotations/build.gradle

```diff
@@ -1,24 +1,10 @@
 apply plugin: 'kotlin'
 
-dependencies {
-    ext.kotlinVersion = '1.4.10'
```

Comment:
I'm surprised by this deletion, even if this builds now, doesn't that mean that when a new version of Kotlin will be available, people might start using the new features and accidentally introduce breaking changes that won't compile on other machines using the old one? (which I assume is the reason why the Kotlin version was fixed in the first place)?

Reply:
This was used in `implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"` dependency declaration. When I removed the dependency, I removed the property as well. And when I [reverted the removal of the dependency](https://github.com/wordpress-mobile/WordPress-Android/pull/14797#discussion_r647969582), I've also reverted this property setter. Hope that makes sense!